lib/config.js

```diff
@@ -29,15 +36,16 @@ if (argv.config) {
   if (typeof argv.config === 'string') {
     argv.config = [argv.config];
   }
-  for (var i in argv.config) {
-    loader.add(path.resolve(argv.config[i]));
-  }
+
+  argv.config.forEach(function(val) {
+    loader.add(path.resolve(val));
+  });
 }
 
 var setOptions = {};
 var key = null;
 for (key in yargs.argv) {
-  if (yargs.argv[key] !== undefined) {
+  if (!_.isUndefined(yargs.argv[key])) {
```

Comment:
@lliss but the thing is, it's not passing our linting setup _correctly_. The point of the linting rule is to ensure that you properly check the object has the property you're looping over, and you aren't instead looping over an inherited property. The lodash function does not do that. The lodash function is doing the same check it was originally doing (i.e. simply a type check).

`hasOwnProperty` does exactly what the linting rule is guarding against. if you still want to use lodash for some reason, read about [_.has](https://lodash.com/docs#has). In this case, the lodash function doesn't "prevent problems" in using the native functions. It just expands on them.